test/unit-test/core_sntp_client_utest.c

```diff
@@ -953,6 +1075,7 @@ void test_ReceiveTimeResponse_Nominal()
 
     /* Test when server response is received without server validation. */
     context.authIntf.validateServerAuth = NULL;       /* Remove the authentication interface from the context. */
+    context.authIntf.generateClientAuth = NULL;
```

Comment:
Missing comment here but personally feel like the comments can be a little superfluous at times

Reply:
The comment on line 1077 applies to line 1078 as well.